package.json

```diff
@@ -53,6 +53,7 @@
     "babel-plugin-transform-es2015-modules-commonjs": "^6.26.2",
     "colors": "^1.4.0",
     "commitizen": "^3.1.1",
+    "core-js": "^3.5.0",
```

Comment:
This should be a dependency even though it's just part of a config for a preset in `babel-loader`.

https://babeljs.io/docs/en/babel-preset-env#usebuiltins